### PR TITLE
Rework responsibility model for clarity

### PR DESCRIPTION
**Pull request by ba-edb**
## What Changed?

We've received some feedback on the clarity of the BigAnimal shared responsibility document. This changeset rewrites several points for clarity and correctness.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx [hunk 1]
```diff
@@ -1,37 +1,40 @@
 ---
-title: "Responsibility model"
+title: "Shared responsibilities"
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx:2]:
```suggestion
title: "Shared security responsibilities"
```

File: product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
--   You are responsible for data modeling, query performance, and scaling the cluster to meet your performance needs. 
+-   EDB is responsible for deploying clusters with the infrastructure you choose, and managing and monitoring these infrastructure resources. 
+-   You are responsible for data modeling, query design, and scaling the cluster to meet your performance needs. 
 
-## Scaling
+## Deploying and scaling
 
--   EDB is responsible for managing and monitoring the underlying infrastructure.
--   You are responsible for choosing the appropriate resources for your workload, including instance type, storage, and connections. You are also responsible for managing your Azure resource limits to ensure the underlying infrastructure can scale.
+-   EDB is responsible for deploying, managing and monitoring the underlying infrastructure supporting your clusters.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx:21]:
```suggestion
-   EDB is responsible for deploying, managing, and monitoring the underlying infrastructure supporting your clusters.
```

File: product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx [hunk 3]
```diff
@@ -1,37 +1,40 @@
...
+-   EDB is responsible for data encryption at rest for both backups and live data.
+-   EDB is responsible for data encryption in transit for both intra-cluster traffic and traffic between clusters and backup storage.
+-   You are responsible for data encryption in transit between your applications and your cluster: BigAnimal clusters support, but *do not require*, `verify-full` TLS connections.
+-   You are responsible for application-level encryption to protect particularly sensitive data from unauthorized access by your authorized users and applications.
 
 ## Credential management
 
--   EDB is responsible for making credentials available to you.
--   You are responsible for managing and securing your passwords, both for BigAnimal and for your database.
+-   EDB is responsible for securely managing your edb_admin credential. The edb_admin credential will never be stored in plaintext.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx:39]:
```suggestion
-   EDB is responsible for securely managing your edb_admin credential. The edb_admin credential is never stored in plaintext.
```

File: product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx (hunk 1 above)
Comment (ba-edb) [reply product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx:2]: We shouldn't limit this to security. Performance is an important part of the shared responsibility model, for example.

**Review by drothery-edb — COMMENTED** [review]
LGTM - want me to merge or do you want to wait for Aislinn to review?

**Comment by ba-edb** [comment]
I'm good if you merge it, thanks. /cc @aswright491

**Pull request merged**